src/routes/routes.ts

```diff
@@ -19,3 +25,5 @@ routes.get(ROOT_URI, startPageController.renderView);
 routes.post(ROOT_URI, startPageController.start);
 routes.get(APPLICANT_DETAILS_PAGE_URI, applicantDetailsController.renderView);
 routes.post(APPLICANT_DETAILS_PAGE_URI, applicantDetailsController.processForm);
+routes.get(ADDRESS_TO_REMOVE_PAGE_URI, addressToRemoveController.renderView);
+routes.post(ADDRESS_TO_REMOVE_PAGE_URI, addressToRemoveController.processForm);
```

Comment:
I'd prefer if we had spaces between each page for readability. If we were to use Inversify we could probably avoid this whole routes list, but we can discuss that in the upcoming meetings

```suggestion
routes.post(ROOT_URI, startPageController.start);

routes.get(APPLICANT_DETAILS_PAGE_URI, applicantDetailsController.renderView);
routes.post(APPLICANT_DETAILS_PAGE_URI, applicantDetailsController.processForm);

routes.get(ADDRESS_TO_REMOVE_PAGE_URI, addressToRemoveController.renderView);
routes.post(ADDRESS_TO_REMOVE_PAGE_URI, addressToRemoveController.processForm);
```